src/styles/_addbook.scss

```diff
@@ -0,0 +1,61 @@
+.book-container {
+  padding: 3%;
+
+  .books {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+  }
+
+  .horizontal-divider {
+    height: 0.2rem;
+    background-color: #f5f6f5;
+    width: 100%;
+    margin: 2.5rem 0 1.813rem;
+  }
+}
+
+.Add-book-container {
+  .add-title {
+    font-size: 1.25rem;
+    font-weight: 700;
+    letter-spacing: -0.18px;
+    color: #888;
+  }
+
+  .add-book {
```

Comment:
:red_circle: Kindly make sure you move the add_book button to the far right to match the Figma Design. You can add a `justify-content: space-between  rule to this class  to fix this issue

Your Design
![Screenshot from 2023-01-27 15-19-48](https://user-images.githubusercontent.com/26736582/215098411-922dd56d-b7fc-4a57-8866-c0fbad9bcc1c.png)

Template
![Screenshot from 2023-01-27 15-20-26](https://user-images.githubusercontent.com/26736582/215098447-18eb3e19-dc70-4b5a-803b-e868c70d51cf.png)

Reply:
FIxed the button position